src/ring_vrf_impl.rs

```diff
@@ -192,30 +193,38 @@ impl GenerateVerifiable for BandersnatchVrfVerifiable {
 		// This doesn't require the whole kzg. Thus is more appropriate if used on-chain
 		// Is a bit slower as it requires to recompute piop_params, but still in the order of ms
 		let ring_verifier =
-			bandersnatch_vrfs::ring::make_ring_verifier(members.0.clone(), DOMAIN_SIZE);
+			bandersnatch::RingProofParams::verifier_no_context(members.0.clone(), RING_SIZE);
 
-		let vrf_input = Message {
-			domain: VRF_INPUT_DOMAIN,
-			message: context,
-		}
-		.into_vrf_input();
+		let input_msg = [VRF_INPUT_DOMAIN, context].concat();
+		let input = bandersnatch::Input::new(&input_msg[..]).expect("H2C can't fail here");
 
-		let ring_signature =
+		let signature =
 			RingVrfSignature::deserialize_compressed(proof.as_slice()).map_err(|_| ())?;
 
-		let ios = RingVerifier(&ring_verifier)
-			.verify_ring_vrf(message, core::iter::once(vrf_input), &ring_signature)
-			.map_err(|_| ())?;
+		bandersnatch::Public::verify(
+			input,
+			signature.output,
+			message,
+			&signature.proof,
+			&ring_verifier,
+		)
+		.map_err(|_| ())?;
 
-		let alias: Alias = ios[0].vrf_output_bytes(VRF_OUTPUT_DOMAIN);
-		Ok(alias)
+		Ok(make_alias(&signature.output))
 	}
 
+	// TODO @davxy: maybe implement a plain Schnorr signature for Bandersnatch?
```

Comment:
@Zebedeusz what's the usage of the plain signature in Proof of person hood project?

Reply:
Currently it's used in tests and benchmarks only.
So we expect the users to provide their signature for some operations or in some cases.
Do you need more details?